src/contractTest/java/uk/gov/hmcts/reform/sscs/consumer/HearingPostConsumerTest.java

```diff
@@ -0,0 +1,106 @@
+import uk.gov.hmcts.reform.sscs.service.HmcHearingApi;
+import uk.gov.hmcts.reform.sscs.utility.HearingResponsePactUtil;
+
+@ExtendWith(SpringExtension.class)
+@ExtendWith(PactConsumerTestExt.class)
+@EnableFeignClients(basePackages = {"uk.gov.hmcts.reform.sscs.service"})
+@ActiveProfiles("contract")
+@SpringBootTest
+@PactTestFor(port = "10000")
+@PactFolder("build/pacts")
+public class HearingPostConsumerTest extends BasePactTesting {
+
+    private static final String PATH_HEARING = "/hearing";
+    private static final String FIELD_STATUS = "status";
+    private static final String BAD_REQUEST = "BAD_REQUEST";
+    private static final String FIELD_MESSAGE = "message";
+    private static final String FIELD_ERRORS = "errors";
+
+    @Autowired
+    private HmcHearingApi hmcHearingApi;
+
+    HearingRequestPayload validRequest = generateHearingRequest();
+    String jsonValidRequest = toJsonString(validRequest);
+    HearingRequestPayload invalidRequest = generateInvalidHearingRequest();
+    String jsonInvalidRequest = toJsonString(invalidRequest);
+
+    @Pact(consumer = CONSUMER_NAME)
+    public RequestResponsePact createHearingRequestForValidRequest(PactDslWithProvider builder) {
+        return builder.given(CONSUMER_NAME + " successfully creating hearing request ").uponReceiving(
+            "Request to create hearing request to save details")
+            .path(PATH_HEARING).method(HttpMethod.POST.toString()).body(
+            jsonValidRequest).headers(headers).willRespondWith()
+            .status(HttpStatus.OK.value()).body(
+            HearingResponsePactUtil
+                .generatePostHearingsJsonBody(MSG_200_POST_HEARING))
+            .toPact();
+    }
+
+    @Pact(consumer = CONSUMER_NAME)
+    public RequestResponsePact validationErrorFromPostHearing(PactDslWithProvider builder) {
+        return builder.given(CONSUMER_NAME
+                                 + " throws validation error while trying to create hearing")
+                .uponReceiving("Request to CREATE hearing for invalid hearing request")
+                    .path(PATH_HEARING).method(HttpMethod.POST.toString())
+            .body(jsonInvalidRequest).headers(headers)
+            .willRespondWith().status(HttpStatus.BAD_REQUEST.value())
+            .body(new PactDslJsonBody().stringType(FIELD_MESSAGE, MSG_400_POST_HEARING)
+                      .stringValue(FIELD_STATUS, BAD_REQUEST)
+                          .eachLike(FIELD_ERRORS, 1)
+                      .closeArray())
+            .toPact();
+    }
+
+    @Test
+    @PactTestFor(pactMethod = "createHearingRequestForValidRequest")
+    public void shouldSuccessfullyPostHearingRequest() throws JsonProcessingException {
+        HearingResponse hearingResponse = hmcHearingApi.createHearingRequest(
+            IDAM_OAUTH2_TOKEN,
+            SERVICE_AUTHORIZATION_TOKEN,
+            generateHearingRequest()
+        );
+
+        Assertions.assertTrue(hearingResponse.getHearingRequestId() > 0);
```

Comment:
(Minor) Would be good to use assertJ's assertion library -i t has some really good built in assertion methods.